README.md

```diff
@@ -1,9 +1,66 @@
-# triton-docker
+# Triton Docker CLI helper
 
-Docker CLI helper for Triton
+This script installs known-good, tested versions of the Docker (now Moby) and Docker Compose CLI tools for use with Triton.
```

Comment:
Maybe link out now Moby to the [related PR release](https://blog.docker.com/2017/04/introducing-the-moby-project/)? Perhaps it's because I wasn't at DockerCon, but this was new information.

That said, it's not an evergreen decision, so it would make sense to update as the name of the project updates.